docs/vue.md

```diff
@@ -20,118 +20,314 @@ To get started, load either the production or development version of Vue in your
 
 ```markdown
+<!-- Hide in docsify, show elsewhere (e.g. GitHub) -->
+<p v-if="false">Text for GitHub</p>
+
+<!-- Sequenced content (i.e. loop)-->
 <ul>
-  <li v-for="i in 3">{{ i }}</li>
+  <li v-for="i in 3">Item {{ i }}</li>
 </ul>
+
+<!-- JavaScript expressions -->
+<p>2 + 2 = {{ 2 + 2 }}</p>
 ```
 
-The HTML above will render the following:
+<output data-lang="output">
+  <p v-if="false">Text for GitHub</p>
 
-<ul>
-  <li v-for="i in 3">{{ i }}</li>
-</ul>
+  <ul>
+    <li v-for="i in 3">Item {{ i }}</li>
+  </ul>
 
-## Advanced usage
+  <p>2 + 2 = {{ 2 + 2 }}</p>
+</output>
 
-Vue components and templates that require `data`, `methods`, computed properties, lifecycle hooks, etc. require manually creating a new `Vue()` instance within a `<script>` tag in your markdown.
+[View output on GitHub](https://github.com/docsifyjs/docsify/blob/develop/docs/vue.md#template-syntax)
 
-<!-- prettier-ignore-start -->
+Vue content becomes more interesting when [data](#data), [computed properties](#computed-properties), [methods](#methods), and [lifecycle hooks](#lifecycle-hooks) are used. These options can be specified as [global options](#global-options), [mount options](#mount-options), or within [components](#components).
+
+### Data
 
+```js
+{
+  data() {
+    return {
+      message: 'Hello, World!'
+    };
+  }
+}
+```
+
+<!-- prettier-ignore-start -->
 ```markdown
-<div id="example-1">
-  <p>{{ message }}</p>
+<!-- Show message in docsify, show "{{ message }}" elsewhere (e.g. GitHub)  -->
+{{ message }}
 
-  <button v-on:click="hello">Say Hello</button>
+<!-- Show message in docsify, hide elsewhere (e.g. GitHub)  -->
+<p v-text="message"></p>
 
-  <button v-on:click="counter -= 1">-</button>
-  {{ counter }}
-  <button v-on:click="counter += 1">+</button>
-</div>
+<!-- Show message in docsify, show text elsewhere (e.g. GitHub)  -->
+<p v-text="message">Text for GitHub</p>
 ```
-
 <!-- prettier-ignore-end -->
 
-#### Vue 2.x
+<output data-lang="output">
 
-```markdown
-<script>
-  new Vue({
-    el: "#example-1",
-    data: function() {
-      return {
-        counter: 0,
-        message: "Hello, World!"
-      };
-    },
-    methods: {
-      hello: function() {
-        alert(this.message);
+{{ message }}
+
+  <p v-text="message"></p>
+  <p v-text="message">Text for GitHub</p>
+</output>
+
+[View output on GitHub](https://github.com/docsifyjs/docsify/blob/develop/docs/vue.md#data)
+
+### Computed properties
+
+```js
+{
+  computed: {
+    timeOfDay() {
+      const date = new Date();
+      const hours = date.getHours();
+
+      if (hours < 12) {
+        return 'morning';
+      }
+      else if (hours < 18) {
+        return 'afternoon';
+      }
+      else {
+        return 'evening'
       }
     }
-  });
-</script>
+  },
+}
 ```
 
-#### Vue 3.x
+```markdown
+Good {{ timeOfDay }}!
+```
+
+<output data-lang="output">
```

Comment:
What does `data-lang="output"` do?

Reply:
Oooh, nevermind, I see: the `<output>` element is not the result that the user will see in their DOM, but rather it is used here in the docs to show what the result will _be like_.

Reply:
Yep! 😄